### PR TITLE
Gradle updates

### PR DESCRIPTION
**Pull request by cthigh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 /target/
```
Comment (mattbsox) [inline .gitignore:1]: We should add in the eclipse generated files like .project, .classpath, and .settings to the .gitignore.

File: .settings/.jsdtscope [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattbsox) [inline .settings/.jsdtscope]: remove this eclipse file

File: .gitignore (hunk 1 above)
Comment (mattbsox) [reply .gitignore:1]: We should also remove all these eclipse files from the PR

**Review by mattbsox — CHANGES_REQUESTED** [review]

**Review by dshimo — CHANGES_REQUESTED** [review]
We should remove most of these Eclipse-generated files.

Also there's a comment block in `build.gradle` we could remove

File: README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ This sample requires that you create some resources before you deploy the applic
   ```bash
     ./wsadmin.sh -f addAuthAlias.py user_name password
    ```
-2. If you are running the sample in WebSphere Liberty Profile follow the intructions in the sections below.
+2. If you are running the sample in WebSphere Liberty Profile follow the instructions in the sections below.
```
Comment (cthigh) [inline README.md]: change `WebSphere Liberty Profile` to just `Liberty`

File: README.md [hunk 4]
```diff
@@ -61,6 +61,30 @@ Use the following steps to run the application with Maven:
 
 Once the server is running, the application will be available under [http://localhost:9080/sample.javaee7.jms/](http://localhost:9080/sample.javaee7.jms/).
 
+## Running with Gradle
+
+This project can be built with [Apache Gradle]. The project uses [Liberty Gradle Plug-in](https://github.com/WASdev/ci.gradle) to automatically download and install Liberty with Java EE7 Full Platform runtime from [Maven Central]. Liberty Gradle Plug-in is also used to create, configure, and run the application on the Liberty server. 
+
+Use the following steps to run the application with Gradle:
+
+1. Execute full Gradle build. This will cause Liberty Gradle Plug-in to download and install Liberty profile server.
```
Comment (cthigh) [inline README.md:70]: change to:

Execute the full Gradle build. The Liberty Gradle Plug-in will download and install the Liberty runtime and create the server.

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    testCompile group: 'org.glassfish', name: 'javax.json', version:'1.0.4'
+    providedCompile group: 'javax.servlet', name: 'javax.servlet-api', version:'3.1.0'
+    providedCompile group: 'org.apache.geronimo.specs', name: 'geronimo-servlet_3.0_spec', version:'1.0'
+    providedCompile group: 'javax.jms', name: 'javax.jms-api', version:'2.0'
+    libertyRuntime group: 'com.ibm.websphere.appserver.runtime', name: 'wlp-javaee7', version: '17.0.0.2'
+}
+
+ext {
+    testServerHttpPort = '9080'
+    testServerHttpsPort = '9443'
```
Comment (cthigh) [inline build.gradle]: you can remove the quotes from the port numbers.

File: build.gradle [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+// how to do profile equivalent ??????
+
+//    <profiles>
+//        <profile>
+//            <id>runnable</id>
+//            <properties>
+//                <packaging.type>runnable</packaging.type>
+//            </properties>
+//        </profile>
+//    </profiles>
```
Comment (cthigh) [inline build.gradle]: Let's go ahead and remove this profile stuff.

File: .classpath [hunk 7]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattbsox) [inline .classpath]: remove this file

File: .settings/org.eclipse.buildship.core.prefs [hunk 8]
```diff
@@ -1,3 +1,3 @@
-#Mon Oct 02 15:36:01 CDT 2017
+#Mon Oct 09 12:54:30 CDT 2017
```
Comment (mattbsox) [inline .settings/org.eclipse.buildship.core.prefs]: remove this file

Comment (mattbsox) [reply .settings/org.eclipse.buildship.core.prefs]: and any other .settings files

File: .classpath [hunk 9]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattbsox) [inline .classpath]: remove this file

File: .factorypath [hunk 10]
```diff
@@ -0,0 +1,6 @@
+<factorypath>
```
Comment (mattbsox) [inline .factorypath]: remove this file

File: .project [hunk 11]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattbsox) [inline .project]: remove this file

File: .settings/com.ibm.etools.common.migration.prefs [hunk 12]
```diff
@@ -0,0 +1,3 @@
+#Mon Oct 02 15:36:44 CDT 2017
```
Comment (mattbsox) [inline .settings/com.ibm.etools.common.migration.prefs]: remove this file

File: .settings/org.eclipse.buildship.core.prefs [hunk 13]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (mattbsox) [inline .settings/org.eclipse.buildship.core.prefs]: remove this file

File: .settings/org.eclipse.core.resources.prefs [hunk 14]
```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```
Comment (mattbsox) [inline .settings/org.eclipse.core.resources.prefs]: remove this file

File: .settings/org.eclipse.jdt.apt.core.prefs [hunk 15]
```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```
Comment (mattbsox) [inline .settings/org.eclipse.jdt.apt.core.prefs]: remove this file

File: .settings/org.eclipse.jdt.core.prefs [hunk 16]
```diff
@@ -0,0 +1,9 @@
+eclipse.preferences.version=1
```
Comment (mattbsox) [inline .settings/org.eclipse.jdt.core.prefs]: remove this file

File: .settings/org.eclipse.m2e.core.prefs [hunk 17]
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```
Comment (mattbsox) [inline .settings/org.eclipse.m2e.core.prefs]: remove this file

File: .settings/org.eclipse.wst.common.component [hunk 18]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?><project-modules id="moduleCoreId" project-version="1.5.0">
```
Comment (mattbsox) [inline .settings/org.eclipse.wst.common.component]: remove this file

File: .settings/org.eclipse.wst.common.project.facet.core.prefs.xml [hunk 19]
```diff
@@ -0,0 +1,7 @@
+<root>
```
Comment (mattbsox) [inline .settings/org.eclipse.wst.common.project.facet.core.prefs.xml]: remove this file

File: .settings/org.eclipse.wst.common.project.facet.core.xml [hunk 20]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattbsox) [inline .settings/org.eclipse.wst.common.project.facet.core.xml]: remove this file

File: .settings/org.eclipse.wst.jsdt.ui.superType.container [hunk 21]
```diff
@@ -0,0 +1 @@
+org.eclipse.wst.jsdt.launching.baseBrowserLibrary
```
Comment (mattbsox) [inline .settings/org.eclipse.wst.jsdt.ui.superType.container]: remove this file

File: .settings/org.eclipse.wst.jsdt.ui.superType.name [hunk 22]
```diff
@@ -0,0 +1 @@
+Window
```
Comment (mattbsox) [inline .settings/org.eclipse.wst.jsdt.ui.superType.name]: remove this file

File: .settings/org.eclipse.wst.validation.prefs [hunk 23]
```diff
@@ -0,0 +1,2 @@
+disabled=06target
```
Comment (mattbsox) [inline .settings/org.eclipse.wst.validation.prefs]: remove this file

File: build.gradle [hunk 24]
```diff
@@ -0,0 +1,97 @@
+apply plugin: 'war'
+apply plugin: 'liberty'
+
+group = 'liberty.samples'
+version = '1'
+
+description = "Liberty JDBC Gradle example"
+
+sourceCompatibility = 1.7
```
Comment (mattbsox) [inline build.gradle:9]: Set these 2 compatibility properties to 1.8

File: pom.xml [hunk 25]
```diff
@@ -51,7 +66,7 @@
             <plugin>
                 <groupId>net.wasdev.wlp.maven.plugins</groupId>
```
Comment (mattbsox) [inline pom.xml:67]: The runtime version can be set to 17.0.0.2

**Review by mattbsox — CHANGES_REQUESTED** [review]

**Pull request merged**